plugins/alerts.js

```diff
@@ -55,6 +70,19 @@ async function alerts (app, _opts) {
         healthCache.splice(0, validIndex)
       }
 
+      // Track worker start time if not already tracked. This handles cases
+      // where the worker started before the plugin was initialized.
+      if (!workerStartTimes.has(workerId)) {
+        workerStartTimes.set(workerId, timestamp)
```

Comment:
I would use the plugin start time then as a default. This might be dangerous especially for the demo as it would ignore the first 30s of a traffic spike.